### PR TITLE
RCHAIN-3628 add test_deploy_invalid_contract

### PR DESCRIPTION
**Pull request by zsluedem**
### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-3628

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Comment by zsluedem** [comment]
bors try

**Comment by zsluedem** [comment]
bors try

File: integration-tests/test/test_propose.py [hunk 1]
```diff
@@ -51,3 +52,19 @@ def test_find_block_by_deploy_id(started_standalone_bootstrap_node: Node, random
...
+
+
+def test_deploy_invalid_contract(started_standalone_bootstrap_node: Node) -> None:
+    shutil.copyfile('resources/invalid.rho', os.path.join(started_standalone_bootstrap_node.local_deploy_dir, 'invalid.rho'))
+    invalid_contract_path = os.path.join(started_standalone_bootstrap_node.remote_deploy_dir, 'invalid.rho')
+
+
+    with pytest.raises(NonZeroExitCodeError) as exc:
+        started_standalone_bootstrap_node.deploy(invalid_contract_path, DEPLOY_KEY)
+    assert "Parsing error" in exc.value.output
```
Comment (adaszko) [inline integration-tests/test/test_propose.py]: Perhaps it's worth introducing a separate exception class `ParsingError`?

**Review by adaszko — APPROVED** [review]
👍

**Review by ghost — APPROVED** [review]

**Pull request merged**